src/main/java/com/umc/networkingService/domain/test/controller/TestController.java

```diff
@@ -32,4 +34,55 @@ public String example() {
     public String ignore() {
         return "무시되는 API";
     }
+
+    private final TestService tempService;
+
+    @GetMapping("/test")
+    public BaseResponse<TestResponse.TempTestDTO> testAPI() {
+
+        return BaseResponse.onSuccess(TestResponse.TempTestDTO.builder().testString("This is Test!").build());
+    /*
+    {
+      "timestamp": "2024-01-12T17:19:02.0172718",
+      "code": "COMMON200",
+      "message": "요청에 성공하였습니다.",
+      "result": {
+        "testString": "This is Test!"
+      }
+    }
+     */
+    }
+
+    @GetMapping("/exception")
+    public BaseResponse<TestResponse.TempExceptionDTO> exceptionAPI(@RequestParam Integer flag) {
+        tempService.CheckFlag(flag);
+        return BaseResponse.onSuccess(TestResponse.TempExceptionDTO.builder()
+                .flag(flag)
+                .build());
+    }
+    /*
+    {
+      "timestamp": "2024-01-12T17:27:10.1903229",
+      "code": "TEMP4001",
+      "message": "예외처리 테스트입니다."
+    }
+     */
+
+    @PostMapping("/test-validation")
```

Comment:
테스트 api이긴 한데 주석을 다는 것보다 swagger에 api에 대해 약간의 설명으로 바꾸는게 좋을 거 같습니다.
```suggestion
    @Operation(summary = "유효성 검사 API", description = "요청 객체의 유효성을 검사하는 API입니다.")
    @ApiResponses(value = {
            @ApiResponse(responseCode = "200", description = "성공"),
            @ApiResponse(responseCode = "COMMON400", description = "testString가 공백일 경우")
    })
    @PostMapping("/test-validation")
```